sqlmesh/core/engine_adapter/spark.py

```diff
@@ -47,11 +47,17 @@ def _ensure_pyspark_df(self, generic_df: DF) -> PySparkDataFrame:
             raise SQLMeshError("Ensure PySpark DF can only be run on a PySpark or Pandas DataFrame")
         return self.spark.createDataFrame(df)
 
-    def fetchdf(self, query: t.Union[exp.Expression, str]) -> pd.DataFrame:
-        return self.fetch_pyspark_df(query).toPandas()
+    def fetchdf(
+        self, query: t.Union[exp.Expression, str], normalize_identifiers: bool = True
```

Comment:
Should this actually default to `False` since this is a user facing API?

Reply:
Good point, will update. Thanks!

Reply:
done